js/modules/data.d.ts

```diff
@@ -1,3 +1,8 @@
 export function searchMessages(query: string): Promise<Array<any>>;
 export function searchConversations(query: string): Promise<Array<any>>;
 export function getPrimaryDeviceFor(pubKey: string): Promise<string | null>;
+
+export function getMessagesByConversation(
+  conversationId: string,
+  destructurer: any
+): Promise<Array<any>>;
```

Comment:
In the future we should create a typescript definition for a message so we can pass it in here instead of `any`